src/components/global/Footer.jsx

```diff
@@ -79,41 +74,50 @@ const renderSocialLinks = socialMediaLinks.map(({ ariaLabel, href, icon }) => (
   </Link>
 ));
 
-// maps legalLinks used for placement in stack below
-const renderLegalLinks = legalLinks.map((link, index) => (
-  <Box key={link.title + index}>
-    <Typography
-      key={link.title}
-      variant="body1"
-      color="tertiary.main"
-      mr={{ sm: '10px' }}
-      sx={{
-        display: 'flex',
-        justifyContent: { xs: 'center', sm: 'flex-end' },
-        p: { xs: '10px', sm: '5px' }
-      }}
-    >
-      {link.text ?? null}
-      <Link
-        component={ReactRouterLink}
-        to={link.href}
-        target="_blank"
-        rel="noopener noreferrer"
-        underline="none"
-        aria-label={`${link.title}`}
-        ml={link.ml ?? null}
+// Maps legalLinks used for placement in stack below
+const renderLegalLinks = (navigate) => {
+  return legalLinks.map((link, index) => (
+    <Box key={link.title + index}>
+      <Typography
+        variant="body1"
+        color="tertiary.main"
+        mr={{ sm: '10px' }}
         sx={{
-          color: '#000',
-          '&:hover': {
-            color: 'primary.main'
-          }
+          display: 'flex',
+          justifyContent: { xs: 'center', sm: 'flex-end' },
+          p: { xs: '10px', sm: '5px' }
         }}
       >
-        {link.title}
-      </Link>
-    </Typography>
-  </Box>
-));
+        {link.text ?? null}
+        <Link
+          href={link.href}
+          target="_blank"
+          rel="noopener noreferrer"
+          underline="none"
+          aria-label={`${link.title}`}
+          ml={link.ml ?? null}
+          sx={{
+            color: '#000',
+            '&:hover': {
+              color: 'primary.main'
+            }
+          }}
+          onClick={
+            link.title === 'CODE PDX'
+              ? (event) => {
+                  event.preventDefault();
+                  navigate('/');
```

Comment:
`navigate('/');` can be removed, the scroll to top takes care of that links functionality since it stays on same page.